packages/angular/build/src/tools/esbuild/budget-stats.ts

```diff
@@ -61,6 +61,10 @@ export function generateBudgetStats(
   // Add component styles from metafile
   // TODO: Provide this information directly from the AOT compiler
   for (const entry of Object.values(metafile.outputs)) {
+    const file = Object.keys(entry.inputs)[0];
+    if (!file) {
+      continue;
+    }
```

Comment:
```suggestion
  for (const [file, entry] of Object.entries(metafile.outputs)) {
    if (!file.endsWith('.css')) {
      continue;
    }
```
Only the component stylesheets should be handled here.